pkgs/tools/admin/lxd/ui.nix

```
@@ -0,0 +1,42 @@
+{ mkYarnPackage
+, fetchFromGitHub
+, fetchYarnDeps
+, lib
+, ...
+}:
+mkYarnPackage rec {
+  pname = "lxd-ui";
+  version = "0.0.1";
```

There is not a released version, so this should be an unstable release.

```suggestion
  version = "unstable-2023-07-03";
```